db/models.py

```diff
@@ -50,3 +56,73 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        to=AUTH_USER_MODEL,
```

Comment:
Ensure that `AUTH_USER_MODEL` is correctly set in your Django settings to point to the custom `User` model. This is important for maintaining the integrity of the foreign key relationship in the `Order` model.